internal/handler/hellohandler.go

```diff
@@ -0,0 +1,15 @@
+package handler
+
+import (
+    "fmt"
+    "net/http"
+)
+
+func HelloHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
Same as above, do we need "Handler" in the function name?

Reply:
Will update this too